asciidoc-common/install/snip-RHVM_repo_steps.adoc

```diff
@@ -0,0 +1,41 @@
+. Enable the required repositories:
+.. Register your system with the Content Delivery Network, entering your Customer Portal user name and password when prompted:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager register
+----
++
+.. Find the `{virt-product-fullname} {engine-name}` subscription pool and record the pool ID:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager list --available
+----
++
+.. Use the pool ID to attach the subscription to the system:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager attach --pool=_pool_id_
+----
++
+.. Configure the repositories:
++
+[options="nowrap" subs="normal"]
+----
+# subscription-manager repos \
+    --disable='*' \
+    --enable=rhel-7-server-rpms \
+    --enable=rhel-7-server-supplementary-rpms \
+    --enable=rhel-7-server-rhv-4.3-manager-rpms \
+    --enable=rhel-7-server-rhv-4-manager-tools-rpms \
+    --enable=rhel-7-server-ansible-2-rpms \
```

Comment:
if this is not already under conditional, above repos are fine also for oVirt, except the rhel-7-server-rhv-4.3-manager-rpms and rhel-7-server-rhv-4-manager-tools-rpms which are not available with a RHEL subscripton.

Reply:
I'm not sure about ansible either

Reply:
This is actually conditionalized already, so it seems like that file is unneeded.